tests/test_gls_fitter.py

```diff
@@ -10,6 +10,7 @@
 from pint import toa
 from pint.fitter import GLSFitter
 from pinttestdata import datadir
+import matplotlib.pyplot as plt
```

Comment:
Does this force our test suite to be dependent on matplotlib? Does this still work okay if tests are running headless? Is there any need for this import?